src/Markdig/Extensions/Tables/PipeTableParser.cs

```diff
@@ -47,6 +47,16 @@ public override bool Match(InlineProcessor processor, ref StringSlice slice)
             return false;
         }
 
+        // Do not treat pipe characters that occur within inline code block as table delimiters
+        // This handles the scenario where we start an inline code block but do a new line with a pipe
+        // `
+        // || Code ||
+        // `
+        if (processor.Inline != null && processor.Inline.ContainsParentOfType<CodeInline>())
```

Comment:
When I was testing your previous PR locally with this example I didn't need checks here (only in `CodeInlineParser`).
Is there a different test case this is needed for?

Reply:
This was the first bit of code I put in so I guess it became unnecessary. Removed it after confirming its not affecting the outcome of things.